lib/signatures.js

```diff
@@ -50,9 +50,16 @@ function getSignatures( key ) {
 }
 
 function getSignatureErrors() {
-	return all().then( function( signatures ) {
-		return validate( signatures ).errors;
-	} );
+	return Promise.all( [
+		module.exports.raw( keys.cla ),
+		module.exports.raw( keys.caa )
+	] )
+		.then( function( signatures ) {
+			return signatures[ 0 ].concat( signatures[ 1 ] );
+		} )
+		.then( function( signatures ) {
+			return validate( signatures );
+		} );
```

Comment:
This method no longer just returns errors as the name implies. This also has a lot of overlap with `getSignatures()`/`all()`; should probably consolidate the logic.